src/manifest/dbtProject.ts

```diff
@@ -518,6 +518,34 @@ export class DBTProject implements Disposable {
     return this.dbtProjectIntegration.getColumnsOfSource(sourceName, tableName);
   }
 
+  async getColumnsValues(model: string, column: string) {
+    try {
+      this.terminal.debug(
+        "getColumnValues",
+        "finding distinct values for column",
+        true,
+        { model, column },
+      );
+      const query = `select ${column} as values from ${model} group by ${column}`;
+      const queryExecution = await this.dbtProjectIntegration.executeSQL(
+        query,
+        1000,
```

Comment:
executeSQL expects limit to be passed, i just set to 1000 for now. Is this ok? or should we make limit as optional in executeSql and ignore limit?

Reply:
We need to set it much lower, set it to 100. We should probably add some wording around that.
The application will crash if we load too much data.